### ISSUE TITLE
Choco Install fails if IE has never been launched 

### ISSUE THREAD

**Issue by pedrogarcia** [MathewSachin/Captura#93]
This is the console output

```
PS C:\Users\phg\Desktop> choco install captura
Chocolatey v0.10.7
Installing the following packages:
captura
By installing you accept licenses for the packages.
Progress: Downloading captura 5.0.0... 100%

captura v5.0.0 [Approved]
captura package files install completed. Performing other installation steps.
The package captura wants to run 'chocolateyinstall.ps1'.
Note: If you don't run this script, the installation will fail.
Note: To confirm automatically next time, use '-y' or consider:
choco feature enable -n allowGlobalConfirmation
Do you want to run the script?([Y]es/[N]o/[P]rint): y

ERROR: The response content cannot be parsed because the Internet Explorer engine is not available, or Interne
t Explorer's first-launch configuration is not complete. Specify the UseBasicParsing parameter and try again.
The install of captura was NOT successful.
Error while running 'C:\ProgramData\chocolatey\lib\captura\tools\chocolateyinstall.ps1'.
 See log for details.

Chocolatey installed 0/1 packages. 1 packages failed.
 See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).

Failures
 - captura (exited -1) - Error while running 'C:\ProgramData\chocolatey\lib\captura\tools\chocolateyinstall.ps1'.
 See log for details.
```
Apparently, the command ```Invoke-WebRequest https://api.github.com/repos/MathewSachin/Captura/releases/latest``` fails if IE has never run. 
Either the parameter "-UseBasicParsing" should added, or user should be prompted to run IE.
Updating captura also fails under the conditions above.

**Comment by MathewSachin** [comment]
Thank you for reporting the error 👍

It is strange that Powershell depends on IE.
I will check `-UseBasicParsing` and other possible options.

**Comment by MathewSachin** [comment]
Seems to be fixed in v5.0.1.
